server/interceptor/interceptor.go

```diff
@@ -335,19 +338,8 @@ func getRequestInfo(req interface{}) (trillian.TreeType, bool, error) {
 }
 
 func getAdminRequestInfo(req interface{}) (bool, bool) {
```

Comment:
Function name no longer describes what it does.

Reply:
Function name is fine, its purpose didn't change,. It just happens that it's looking at a set of 1 request types now.

Reply:
In other words, it no longer deals with an `AdminRequest` it deals with an `UpdateTreeRequest` -- so the function name is misleading.  To put it another way, if someone added a new callsite `ro, ok := getAdminRequestInfo(delTreeReq)` they wouldn't get what they expected and there's no help from the name.

Reply:
It just doesn't list requests that are `doNotIntercept`. If you don't mind, could I punt the intercept refactors to a separate PR?